src/Doctrine/Version.php

```diff
@@ -23,6 +23,7 @@ class Version
     public function __construct(Connection $connection, string $tablePrefix = 'bolt')
     {
         $this->connection = $connection;
+        $tablePrefix = is_array($tablePrefix) ? $tablePrefix['default'] : $tablePrefix;
```

Comment:
`strict_types` is true, so `tablePrefix` can never be an array, right?

Reply:
Yes, you are corect! I updated the code

Reply:
`string|array` won't work. Union types are only allowed in PHP 8 and up.. Perhaps do it like this, then? 

```php
    /**
     * @param string|array $tablePrefix
     */
    public function __construct(Connection $connection, $tablePrefix = 'bolt')
```

Reply:
Of course, pushed...